packages/manager/apps/web/client/app/hosting/multisite/diagnostic/dialog.html

```diff
@@ -0,0 +1,50 @@
+<div data-ng-controller="MultisiteDiagnosticCtrl as ctrl">
+    <div
+        data-wizard
+        data-wizard-on-cancel="resetAction"
+        data-wizard-hide-confirm-button="false"
+        data-wizard-hide-cancel-button="false"
+        data-wizard-cancel-button-text="'hosting_multisite_dialog_close' | translate"
+        data-wizard-title="'hosting_multisite_dialog_title' | translate"
+    >
+        <div data-wizard-step>
+            <p
+                data-ng-if="ctrl.diagnosticState === 'info' && !ctrl.isDnsExternal && !ctrl.isDnsAttachedToNic"
```

Comment:
maybe move 'info' and 'error' in constant file

Reply:
yes I thought but was lazy to do it :)